### DISCUSSION TITLE
Non `index.md` landing page?

### DISCUSSION THREAD

**Discussion by A-Baji** [squidfunk/mkdocs-material#4098] [Q&A] [answered]
I would like my landing page to be a file besides `index.md`. Is this possible?

```
nav:
  - Getting Started: 
    - foo: getting_started/index.md
    - bar: getting_started/getting_started.md
  - Tutorials: tutorials.md
  - Concepts: concepts.md
  - API: api.md
```

How can I make `getting_started/index.md` be the landing page?

**Comment by squidfunk** [comment, marked as answer]
Could you clarify what you want to achieve? You'll definitely need an `index.md` page, otherwise, users will see a 404 when they open your documentation. Other than that, maybe the [`mkdocs-redirects`](https://github.com/mkdocs/mkdocs-redirects) plugin would allow to redirect users from `index.md`.

**Reply by A-Baji** [reply]
I wanted the landing page to be a file besides `index.md`. So in my case I wanted it to be `getting_started/index.md`, but the redirect plugin works well enough, thanks.
